participant/app/root.tsx

```diff
@@ -84,6 +87,32 @@ export default function App() {
   // language, this locale will change and i18next will load the correct
   // translation files
   useChangeLanguage(locale);
+  const matomoUrl = MATOMO_URL_BASE || "http://localhost:8080/";
```

Comment:
Please code defaults for prod and make overrides for dev in `participant/.env.example` and `participant/docker-compose.yml`. So in here, let's just use the imported env var.
```suggestion
```